app/assets/javascripts/discourse/app/templates/application.hbs

```diff
@@ -44,6 +44,8 @@
       {{/if}}
     </div>
 
+    <PluginOutlet @name="before-main-outlet" @connectorTagName="div" />
```

Comment:
Do those new ones need `div` connectors? (others have it for backward compatibility)